src/helpers/PlaneHelper.js

```diff
@@ -0,0 +1,53 @@
+import { Box3 } from '../math/Box3';
+import { Line } from '../objects/Line';
+import { LineBasicMaterial } from '../materials/LineBasicMaterial';
+import { Float32BufferAttribute } from '../core/BufferAttribute';
+import { BufferGeometry } from '../core/BufferGeometry';
+
+/**
+  * @author WestLangley / http://github.com/WestLangley
+  *
+ */
+
+function PlaneHelper( plane, size, hex ) {
+
+	this.type = 'PlaneHelper';
+
+	this.plane = plane;
+
+	this.size = size;
+
+	var color = ( hex !== undefined ) ? hex : 0xffff00;
+
+	if ( size === undefined ) size = 1;
```

Comment:
Oops. `this.size` is used in `update()`. I just have the code in the wrong order. Will fix.